package.json

```diff
@@ -49,6 +49,16 @@
     "configuration": {
       "title": "VS Code Markdown Notes Configuration",
       "properties": {
+        "vscodeMarkdownNotes.noteCompletionConvention": {
+          "type": "string",
+          "enum": [
+            "rawFilename",
+            "noExtension",
+            "toSpaces"
+          ],
+          "default": "rawFilenamenoExtension",
+          "description": "When offering completion options for `wiki-note.md`, get `rawFilename` OR `noExtension` (`wiki-link`) or `toSpaces` (`wiki link`). Default is rawFilenamenoExtension. NB: when using workspaceFilenameConvention, only rawFilename is returned."
```

Comment:
@b3u @lukesmurray @jay-pee @quickfold since you were all pretty active on #4 

This PR finishes up adding support for fuzzy matching of the following variants:

```
[[wiki-link.md]]
[[wiki-link]]
[[wiki link]]
```

The definition lookups / peeks can support all of these, but when provider completion options, we need to have a single result, such that when you type `[[w` you get only one of the above.

Here is what I am thinking for how to expose this option.

1. Lmk if you have any comments / feedback
2. Feel free to check out this branch, run a `npm run vpackage` and install from the vsix (or just run in debug mode).

NB: you may have to blow away your `node_modules` if you do build from source because I am removing `mocha` due to a conflict with `jest` (in the process of switching tests to headless so they are faster and can be run from within vscode, without quitting).

🍻

Reply:
Dude you are my hero! Now finally VSCode is usable for my note taking process. Thanks!